cmake/modules/BuildSPDK.cmake

```diff
@@ -10,7 +10,10 @@ macro(build_spdk)
     find_package(uuid REQUIRED)
   endif()
 
-  find_program (MAKE_EXECUTABLE NAMES make gmake)
+  find_program(MAKE_EXECUTABLE NAMES gmake make)
+  if(NOT MAKE_EXECUTABLE)
```

Comment:
could you apply the same change to BuildDPDK.cmake ?

Reply:
yep sure, pushed changes :)